sdk/ai/azure-ai-client/azure/ai/client/_patch.py

```diff
@@ -185,6 +188,41 @@ def from_connection_string(cls, conn_str: str, credential: "TokenCredential", **
         project_name = parts[3]
         return cls(endpoint, subscription_id, resource_group_name, project_name, credential, **kwargs)
 
+    def upload_file(self, file_path: Union[Path, str, PathLike]) -> str:
+        """Upload a file to the Azure AI Studio project.
+           This method required *azure-ai-ml* to be installed.
+
+        :param file_path: The path to the file to upload.
+        :type file_path: Union[str, Path, PathLike]
+        :return: The asset id of uploaded file.
+        :rtype: str
+        """
+        try:
+            from azure.ai.ml import MLClient
+            from azure.ai.ml.entities import Data
+            from azure.ai.ml.constants import AssetTypes
+        except ImportError:
+            raise ImportError(
+                "azure-ai-ml must be installed to use this function. Please install it using `pip install azure-ai-ml`")
```

Comment:
Same here, can you please document in the error message all packages that are needed?